galleries/users_explain/text/fonts.py

```diff
@@ -196,6 +196,33 @@
 Each glyph of the string is rendered using the first font in the list that
 contains that glyph.
 
+Loading new fonts
+-----------------
+
+All fonts that are installed in your system will become available to the
+`.FontManager`, as well as the default fonts that ship with Matplotlib. The latter
+fonts are located in the directory ``'matplotlib\mpl-data\fonts\ttf'``.
+It is also possible to add fonts to this directory, or to `allocate a custom
+directory
+<https://matplotlib.org/stable/api/font_manager_api.html#matplotlib.font_manager.findSystemFonts>`_
+that contains any fonts you wish to add to the `.FontManager`.
+
+In either case, the `.FontManager` will only find the new fonts after the font cache
+has been rebuilt following their addition . You may need to rebuild the cache
+yourself if your new fonts cannot be found by Matplotlib.
+The cache can be rebuilt by using :func:`matplotlib.font_manager.load_fontmanager`::
+    import matplotlib as mpl
+    mpl.font_manager.load_fontmanager(try_read_cache=False)
```

Comment:
I don't think that this will actually work?  

This will make a new font manager and return it to you, but it will not change the singleton that is used by the rest of the library or update the `findfont` and `get_font_names` held at the module level.

Reply:
My testing procedure for this was: 
- add a new font to `mpl-data\fonts\ttf`
- run a short plot with a `text` instance with `family=new font `
- see that the text doesn't update to the new font after multiple runs
- run the `font_manager.load_fontmanager` code
- run the plot again

Now the text is now set to the new font and a new entry in the font cache is present. The `findfont` and `get_font_names` methods both return the new font.
I did notice however, that the before and after fontmanager instances were the same object with the same ID.

Reply:
... ok, I do not understand why that works 🤯

Reply:
ok, question: are you running this all in the same process or each in its own Python process?

Reply:
Each in their own separate process.